Type signatures causing errors in build process

Since I've made dependency updates im facing errors caused by type definition file.


Installed version(s):

````
    "@editorjs/checklist": "^1.6.0",
    "@editorjs/code": "^2.9.0",
    "@editorjs/editorjs": "^2.30.5",
    "@editorjs/header": "^2.8.7",
    "@editorjs/list": "^1.10.0",
    "@editorjs/marker": "^1.4.0",
    "@editorjs/quote": "^2.7.2",
    "@editorjs/table": "^2.4.1",
    "editorjs-tooltip": "^1.2.2",
````

<img width="699" alt="sc" src="https://github.com/user-attachments/assets/620ed746-1943-41e6-af3f-e8c596c80639">

This line within the file `node_modules/@editorjs/table/dist/toolbox.d.ts:29:38` causing the error: 

<img width="671" alt="Bildschirmfoto 2024-08-28 um 23 10 53" src="https://github.com/user-attachments/assets/23c10500-e4a4-49ab-a480-d2b429c6df4e">


and modifying it like this solved it.

<img width="556" alt="Bildschirmfoto 2024-08-28 um 23 11 12" src="https://github.com/user-attachments/assets/73f2e775-007c-415f-ae10-d1168e7e6a6d">

I'm having the same issue, build fails with:

```
node_modules/@editorjs/table/dist/toolbox.d.ts:29:48 - error TS2339: Property 'cssModifier' does not exist on type '{}'.

29     constructor({ api, items, onOpen, onClose, cssModifier }: object);```

Please advise, thanks!

--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "@editorjs/table",
   "description": "Table for Editor.js",
-  "version": "2.4.1",
+  "version": "2.4.2",
   "license": "MIT",
   "repository": "https://github.com/editor-js/table",
   "files": [

--- a/src/toolbox.js
+++ b/src/toolbox.js
@@ -1,6 +1,16 @@
 import Popover from "./utils/popover";
 import * as $ from "./utils/dom";
 import { IconMenuSmall } from "@codexteam/icons";
+
+/**
+ * @typedef {object} PopoverItem
+ * @property {string} label - button text
+ * @property {string} icon - button icon
+ * @property {boolean} confirmationRequired - if true, a confirmation state will be applied on the first click
+ * @property {function} hideIf - if provided, item will be hid, if this method returns true
+ * @property {function} onClick - click callback
+ */
+
 /**
  * Toolbox is a menu for manipulation of rows/cols
  *
@@ -14,11 +24,12 @@ export default class Toolbox {
   /**
    * Creates toolbox buttons and toolbox menus
    *
-   * @param {object} api - Editor.js api
-   * @param {PopoverItem[]} items - Editor.js api
-   * @param {function} onOpen - callback fired when the Popover is opening
-   * @param {function} onClose - callback fired when the Popover is closing
-   * @param {string} [cssModifier] - the modifier for the Toolbox. Allows to add some specific styles.
+   * @param {Object} config
+   * @param {any} config.api - Editor.js api
+   * @param {PopoverItem[]} config.items - Editor.js api
+   * @param {function} config.onOpen - callback fired when the Popover is opening
+   * @param {function} config.onClose - callback fired when the Popover is closing
+   * @param {string} config.cssModifier - the modifier for the Toolbox. Allows to add some specific styles.
    */
   constructor({ api, items, onOpen, onClose, cssModifier = "" }) {
     this.api = api;

